frontend/src/pages/Dashboard.tsx

```
@@ -2,29 +2,38 @@ import { useState, useEffect } from "react";
   const [selectedCrop, setSelectedCrop] = useState("maize");
   const [weatherData, setWeatherData] = useState({ kakamega: null, siaya: null, nairobi: null });
-  const [history, setHistory] = useState([]);
+  const [history, setHistory] = useState<SearchHistoryItem[]>([]);
   const [userId] = useState("default_user"); // Mock userId, replace with auth system
+  
+  // API hooks
+  const adviceApi = useApi<AdviceResponse>();
+  const profileApi = useApi<SearchHistoryItem[]>();
+  
+  // API hooks
+  const adviceApi = useApi<AdviceResponse>();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'adviceApi'. Consider to delete it or rename it.

'adviceApi' is defined here:


(lint/suspicious/noRedeclare)